### PR TITLE
Add `myShopifyDomain` as a deprecated config param to docs

### PR DESCRIPTION
**Pull request by yomexzo**
👀 @richgilbank, @minasmart, @mikkoh 

This only updates the `yuidoc` comments and the hand written docs.

Once this is in, i'd be running to auto generation of the API docs. I did not include that here because it's not been updated for a while and would only be mixing many auto generated files with the actual file changes.

### CODE REVIEW

File: docs/examples/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ To use any of the examples, we first have to set up a Shopify Client in our Java
 $(function() {
   var client = ShopifyBuy.buildClient({
     apiKey: 'your-api-key',
-    myShopifyDomain: 'your-myshopify-domain',
+    domain: 'your-shops-myshopify.com-domain-name',
```
Comment (tessalt) [inline docs/examples/index.md]: let's make it demonstrative.

`yourshopsubdomain.myshopify.com`

**Comment by yomexzo** [comment]
Thanks @tessalt. Updated.

File: docs/examples/index.md (hunk 1 above)
Comment (yomexzo) [inline docs/examples/index.md]: Thanks @tessalt. Updated.

**Comment by mikkoh** [comment]
This looks good to me. @minasmart thoughts?

**Comment by minasmart** [comment]
:shipit:

**Comment by mikkoh** [comment]
@minasmart @yomexzo I think this PR may have slipped through the cracks of not being merged. I just rebase from master. @yomexzo I think it's good to merge since we're working on docs right now.

**Pull request merged**